### PR TITLE
Make StuntmanUser fluent

### LINKED ISSUE #15: Make StuntmanUser more fluent
The current implementation of creating a stuntman user looks like this.

```
new StuntmanUser
{
    Id = "user-2",
    Name = "User 2",
    Claims = new[]
    {
        new Claim("given_name", "Jane"),
        new Claim("family_name", "Doe")
    }
}
```

I'd like to be able to declare a user like this.

```
new StuntmanUser("user-2", "User 2")
  .AddClaim("given_name", "Jane")
  .AddClaim("family_name", "Doe");
```

### PR DESCRIPTION
**Pull request by kendaleiv**
Fixes #15.

This also removes the parameterless constructor on `StuntmanUser`.

### CODE REVIEW

File: src/Core/StuntmanUser.cs [hunk 1]
```diff
@@ -1,19 +1,34 @@
...
 using System.Security.Claims;
 
 namespace RimDev.Stuntman.Core
 {
     public class StuntmanUser
     {
-        public StuntmanUser()
+        public StuntmanUser(string id, string name)
         {
+            if (id == null) throw new ArgumentNullException("id");
```
Comment (khalidabuhakmeh) [inline src/Core/StuntmanUser.cs:11]: should probably check to see if it is empty or null.

File: samples/UsageSample/Startup.cs [hunk 2]
```diff
@@ -13,36 +12,15 @@ public void Configuration(IAppBuilder app)
...
-                            new Claim("family_name", "Smith")
-                        }
-                    },
+                    new StuntmanUser("user-1", "User 1")
+                        .AddClaim("given_name", "John")
+                        .AddClaim("family_name", "Doe"),
+                    new StuntmanUser("user-2", "User 2")
+                        .AddClaim("given_name", "Jane")
+                        .AddClaim("family_name", "Doe"),
+                    new StuntmanUser("user-1", "User 3")
```
Comment (billbogaiv) [inline samples/UsageSample/Startup.cs:21]: Created https://github.com/ritterim/stuntman/issues/18 related to duplicate Ids. Nothing to stop this PR, though.

File: src/Core/StuntmanUser.cs (hunk 1 above)
Comment (kendaleiv) [inline src/Core/StuntmanUser.cs:11]: Addressed in https://github.com/kendaleiv/stuntman/commit/d87c9230caefa736d86ece109e3cfd3144625fd5.

**Comment by khalidabuhakmeh** [comment]
:sparkles:

**Pull request merged**